CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+# Contributing guidelines
+
+## Before contributing
+
+Welcome to [TheAlgorithms/TypeScript](https://github.com/TheAlgorithms/TypeScript)! Before sending your pull requests,
+make sure that you **read the whole guidelines**. If you have any doubt on the contributing guide, please feel free to
+[state it clearly in an issue](https://github.com/TheAlgorithms/TypeScript/issues/new).
+
+## Contributing
+
+### Contributor
+
+We are very happy that you consider implementing algorithms and data structures for others! This repository is
+referenced and used by learners from around the globe. Being one of our contributors, you agree and confirm that:
+
+- You did your work - plagiarism is not allowed.
+  - Any plagiarized work will not be merged.
+- Your work will be distributed under [MIT License](LICENSE) once your pull request is merged.
+- Your submitted work must fulfill our styles and standards.
+
+**New implementation** is welcome! For example, new solutions to a problem, different representations of a graph data
+structure or algorithm designs with different complexity.
+
+**Improving comments** and **writing proper tests** are also highly welcome.
```

Comment:
Which framework is used for testing?

Reply:
For now, none. I think Jest is going to be added in the future.